bindings/nodejs/.github/workflows/CI.yml

```diff
@@ -0,0 +1,495 @@
+name: CI
```

Comment:
This file needs to moved in the root workflows folder and cleaned up. I can totally help with hat.